src/openrct2/network/Network.cpp

```diff
@@ -1113,9 +1148,50 @@ void Network::Server_Send_GAMECMD(uint32 eax, uint32 ebx, uint32 ecx, uint32 edx
     SendPacketToClients(*packet, false, true);
 }
 
+void Network::Client_Send_GAME_ACTION(const GameAction *action)
+{
+    std::unique_ptr<NetworkPacket> packet(NetworkPacket::Allocate());
+
+    uint32_t networkId = 0;
+    networkId = ++_actionId;
+
+    // I know its ugly, want basic functionality for now.
```

Comment:
I'm concerned about this

Reply:
Whats the concern?

Reply:
I mostly meant the removal of const here, but yeah, the issue you write about sounds dangerous as well. I will have to still consider if that's something we can run into or how to mitigate it.

I did try hacking those new game actions a little: started a server with two connected clients, paused the server in a debugger, issued two same actions from clients and unpaused the server. It worked as intended, each client created its own ride.

Reply:
Thats interesting, will also have look later when I'm back home.